monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ComponentsControllerV1.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.monitor.server.controllers;
+
+import org.datacleaner.configuration.DataCleanerConfiguration;
+import org.datacleaner.descriptors.TransformerDescriptor;
+import org.datacleaner.monitor.configuration.*;
+import org.datacleaner.monitor.server.components.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.annotation.PreDestroy;
+import java.util.*;
+
+/**
+ * Controller for DataCleaner components (transformers and analyzers). It enables to use a particular component
+ * and provide the input data separately without any need of the whole job or datastore configuration.
+ * @author k.houzvicka, j.horcicka
+ * @since 8. 7. 2015
+ */
+@Controller
+public class ComponentsControllerV1 implements ComponentsController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ComponentsControllerV1.class);
+    private static final String TENANT = "tenant";
```

Comment:
This confused me, thinking it was a hardcoded tenant-name. But I see it's the parameter name (so no problem as such). But I would rename it to PARAMETER_NAME_TENANT then.